version_control/katalyst_by_codurance/css/objects/_layout.css

```diff
@@ -13,4 +13,10 @@
   --space-2xl: clamp(4.00rem, calc(3.82rem + 0.91vw), 4.50rem);
   --space-3xl: clamp(6.00rem, calc(5.73rem + 1.36vw), 6.75rem);
   
+}
+
+
+.container {
+    max-width: min(var(--content-max-width), 100% - var(--space-s));
```

Comment:
I really like this approach!! Even so, for the right-hand side calculation to work properly I believe it should have a bigger spacing variable or have it multiplied by 2 since it should account for both sides